12_12_2019_HW.py

```diff
@@ -0,0 +1,52 @@
+import requests
+
+from flask import Flask
+from faker import Faker
+
+app = Flask('app')
+
+
+@app.route('/req')
+def requirements():
+    with open('requirements.txt') as f:
+        return '<br>'.join(row for row in f.read().split('\n'))
```

Comment:
`return '<br>'.join(row for row in f.read().split('\n'))` -> `f.read().replace('\n', '<br>')`